### ISSUE TITLE
calcite-slider > layout vertical and horizontal

### ISSUE THREAD

**Issue by SaschaBrunnerCH** [Esri/calcite-components#5522]
### Description

It should be possible to use the slider vertical. Like it is possible with the ArcGIS JS API Slider: https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Slider.html#layout

![image](https://user-images.githubusercontent.com/46531184/196774646-f96e2d46-5727-4f33-80fc-dcfde37b7ef0.png)

### Acceptance Criteria

Possible to configure the layout:
- horizontal
- horizontal-reversed
- vertical
- vertical-reversed

### Relevant Info

See: https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Slider.html#layout

### Which Component

calcite-slider

### Example Use Case

For a sample app we would like to include a slider to select the floor. For that purpose it should be vertical:

Example with the horizontal slider: 
![slider-floor](https://user-images.githubusercontent.com/46531184/196775397-1b56bd7c-6a99-4b9a-84c9-30bfb814227e.gif)

### Esri team

ArcGIS API for JavaScript

**Comment by richiecarmichael** [comment]
This feature is required for:
[#46032](https://devtopia.esri.com/WebGIS/arcgis-js-api/issues/46032) **Range Slider**

The JSAPI Range Slider will be used by MapViewer to assign values to multi-dimensional rasters.

// @driskull @jcfranco

**Comment by ubatsukh** [comment]
@driskull as we discussed yesterday this is a high priority issue for us! MV needs Range Slider for multidimensional image services and the API implementation cannot proceed until the calcite slider has support for this.

**Comment by ashetland** [comment]
Hey folks, see the below for a proposed vertical layout Slider. In addition to `layout=vertical`, this outlines `direction-reversed` and `labels-reversed`. I haven't included range and histogram here, but they'd follow the same logic. Figma file available for more details.
<img width="1624" alt="Slice 1" src="https://user-images.githubusercontent.com/108549080/229635203-c2a97d7b-c798-4f7c-85f0-364a50e85611.png">
<img width="1624" alt="Slice 2" src="https://user-images.githubusercontent.com/108549080/229635218-fa9a2681-5441-4ec7-9b75-f0263d18b603.png">

**Comment by geospatialem** [comment]
Priority has shifted to a medium status, with Maps SDK shifting priority to #6590 and #6591.

**Comment by annelfitz** [comment]
Just chiming in - heard a user request for a vertical slider in calcite at the European DevSummit this year.

**Comment by josercarcamo** [comment]
After meeting with @jcfranco, we've decided to seek an alternative implementation and pause this issue at this time.

**Comment by ashetland** [comment]
[Figma versions](https://www.figma.com/design/vB6xFCAFeW0zKrT4nKlh8Z/Calcite-UI-Kit---Styles?node-id=3921-50) of these we built during the initial development attempt. They should be the point of reference instead of the older design file linked in previous comments.

**Comment by DintaMel** [comment]
Per my conversation with @richiecarmichael and @ubatsukh, this is lower priority given the introduction of the [arcgis-slider](https://developers.arcgis.com/javascript/latest/references/common-components/components/arcgis-slider/). 

They do still want these changes eventually to support the transition from the [arcgis-time-slider](https://developers.arcgis.com/javascript/latest/references/map-components/components/arcgis-time-slider/) to Calcite's Slider.
